Tests/MessagingPush/util/DeviceAttributesProviderTest.swift

```diff
@@ -66,6 +71,36 @@ class DeviceAttributesProviderTest: UnitTest {
             expect.fulfill()
         }
 
+        waitForExpectations()
+    }
+    func test_getDefaultDeviceAttributes_givenSDKWrapperConfig_expectWrapperVersionWithSomeAttributes() {
+        let givenSdkVersion = wrapperMockVersion
+        let givenAppVersion = String.random
+        let givenDeviceLocale = String.random
+        let givenDeviceManufacturer = String.random
+        let expected = [
+            "cio_sdk_version": givenSdkVersion,
+            "app_version": givenAppVersion,
+            "device_locale": givenDeviceLocale,
+            "push_enabled": "true",
+            "device_manufacturer": givenDeviceManufacturer
+        ]
+        deviceInfoMock.underlyingSdkVersion = givenSdkVersion
```

Comment:
Right. Committing your suggestion.

Reply:
Thanks for the change. The test looks like it will need another change. 

Let me make a commit with the change I suggest. You can read it and see what you think.

Reply:
I made a refactor to this tests. Remove some boilerplate and make the test test what we need it to. Continue to make commits changing  things if you do not approve.